src/bin/dual-iir.rs

```diff
@@ -140,7 +140,7 @@ const APP: () = {
 
         loop {
             let sleep = c.resources.mqtt_interface.lock(|interface| {
-                interface.network_stack().poll(clock.current_ms())
+                !interface.network_stack().poll(clock.current_ms())
```

Comment:
`poll() -> true` if the stack has seen activity.

Reply:
Correct, so this is saying `let sleep = !stack_was_updated`, which is semantically equivalent to `let do_not_sleep = stack_was_updated`

Reply:
Ah, right. I got confused by the previous commit removing the negation.